src/main/java/cqltoelm/CQLtoELM.java

```diff
@@ -362,10 +362,9 @@ private void fetchTranslatedLibraries(TranslatedLibrary parentLibrary) throws Cq
                 identifier.setId(include.getPath());
                 identifier.setVersion(include.getVersion());
                 try {
+                    // Todo Rohit test this before merge
```

Comment:
Did this get tested?

Reply:
Yes, I tested, will remove the comment. Thank you